docs/htdocs/info/docs/api/variation/variation_schema.html

```diff
@@ -1726,11 +1732,9 @@ <h2>Metadata tables</h2>
         <li class="sql_schema_species_name">Ovis aries rambouillet</li>
         <li class="sql_schema_species_name">Ovis aries</li>
         <li class="sql_schema_species_name">Pan troglodytes</li>
-        <li class="sql_schema_species_name">Pongo abelii</li>
         <li class="sql_schema_species_name">Rattus norvegicus</li>
         <li class="sql_schema_species_name">Saccharomyces cerevisiae</li>
         <li class="sql_schema_species_name">Sus scrofa</li>
-        <li class="sql_schema_species_name">Taeniopygia guttata</li>
```

Comment:
Why were _Pongo abelii_ and _Taeniopygia guttata_ removed here if they have meta tables?